doc/development.md

```diff
@@ -99,3 +100,11 @@ The key must have access to:
 If starting from a new cloud project, APIs for Cloud Data Store will have to be
 activated and it might be necessary to define indexes with
 `gcloud app deploy index.yaml`.
+
```

Comment:
Let's remove this for now, and implement it in a separate PR. Instead of an extra setup step, it may be even better if we removed the hard failure if the file or the bucket is not there.